package.json

```diff
@@ -22,7 +22,9 @@
   "main": "./lib/service.js",
   "scripts": {
     "start": "heroku-node-settings index.js",
-    "heroku-postbuild": "make auto-version"
+    "bower": "bower",
+    "obt": "obt",
```

Comment:
Could we get rid of these and in the `Makefile` use

```
npx obt@^7 build etc
```

This means that when these commands moves into the shared Makefile the dependent services don't have to explicitly include it in their dependencies.